src/test/java/com/suse/netapi/client/SaltStackClientMockTests.java

```diff
@@ -0,0 +1,111 @@
+package com.suse.saltstack.netapi.client;
+
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.Before;
+
+import static org.junit.Assert.*;
+
+import com.github.restdriver.clientdriver.ClientDriverRule;
+import com.github.restdriver.clientdriver.ClientDriverRequest.Method;
+import com.github.restdriver.clientdriver.ClientDriverRequest;
+
+import static com.github.restdriver.clientdriver.RestClientDriver.giveResponse;
+import static com.github.restdriver.clientdriver.RestClientDriver.onRequestTo;
+
+import com.github.restdriver.clientdriver.exception.ClientDriverFailedExpectationException;
+
+import com.suse.saltstack.netapi.exception.SaltStackException;
```

Comment:
Did the IDE fix the folder-structure/package mismatch by adjusting the package statement and not the folder structure?

My intention was to have the test class in the same package as the class being tested, so these imports weren't necessary.